Snakefile

```diff
@@ -9,6 +9,7 @@ configfile: "configs/config.yaml"
 
 wildcard_constraints:
     countries="[A-Z]{2}",
+    clusters="[0-9]+",
```

Comment:
Hi, @GbotemiB. Thanks for the great contribution! here I have few comments, you will judge how fair are they.

Although, inclusion of clusters as a wildcard might be great. I think that it might be redundant here, because we have here different countries with different amount of clusters. It would be better to read files based on country name and horizon. Because end-user will not know how many clusters are there for each country (they are not equal now). maybe we postpone inclusion of clusters here, but keep it in mind.

Clusters wildcard in `efuels-supply-potentials` repo will be used to read network in future without searching for US_2021 folder. In this repo we might stick to latter. What are your thoughts?

Reply:
I didnt want to include it initially because I thought it wont be needed, but I was considering if there will be a need for it all. But from the user perspective, we would not need it.